### PR TITLE
Refactoring Quantum in HA-mode

### PR DESCRIPTION
**Pull request by xenolog**
- start quantum-metadata-agent on all controllers under pacemaker
- enable network namespaces by default
- change resource ordering in manifests
- some small fixes in ocf-scripts

### CODE REVIEW

File: deployment/puppet/cluster/manifests/init.pp [hunk 1]
```diff
@@ -22,6 +23,15 @@
...
     }
+    file {'ocf-mirantis-path':
+      path=>'/usr/lib/ocf/resource.d/mirantis', 
+      #mode => 755,
+      ensure => directory,
+      recurse => true,
+      owner => root,
+      group => root,
+    } 
+    Package['corosync'] -> File['ocf-mirantis-path']
```
Comment (aglarendil) [inline deployment/puppet/cluster/manifests/init.pp:34]: may be cluster-agents package ?

File: deployment/puppet/quantum/manifests/agents/metadata.pp [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  Package['pacemaker'] -> File['quantum-metadata-agent-ocf']
+
+  # add instructions to nova.conf
+  nova_config { 
+    'service_quantum_metadata_proxy':       value => true; 
+    'quantum_metadata_proxy_shared_secret': value => $shared_secret; 
+  } -> Nova::Generic_service<| title=='api' |>
+
+  quantum_metadata_agent_config {
+    'DEFAULT/debug':                          value => $debug;
```
Comment (bogdando) [inline deployment/puppet/quantum/manifests/agents/metadata.pp:49]: please add one for verbose as well

File: fuel_test/cobbler/test_minimal.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 from fuel_test.config import Config
 from fuel_test.helpers import write_config
 from fuel_test.manifest import Template, Manifest
-from fuel_test.settings import CREATE_SNAPSHOTS, ASTUTE_USE, PUPPET_AGENT_COMMAND
+from fuel_test.settings import CREATE_SNAPSHOTS, ASTUTE_USE, PUPPET_AGENT_COMMAND, QUANTUM_USE_NAMESPACES, TENANT_NETWORK_TYPE
```
Comment (bogdando) [inline fuel_test/cobbler/test_minimal.py:6]: add docs please!

File: fuel_test/cobbler/vm_test_case.py [hunk 4]
```diff
@@ -8,7 +8,7 @@
 from fuel_test.config import Config
 from fuel_test.helpers import tcp_ping, udp_ping, add_to_hosts, await_node_deploy, write_config
 from fuel_test.manifest import Manifest
-from fuel_test.settings import CLEAN, USE_ISO, INTERFACES, PARENT_PROXY, DOMAIN_NAME, CURRENT_PROFILE, PUPPET_MASTER_VERSION
+from fuel_test.settings import CLEAN, USE_ISO, INTERFACES, PARENT_PROXY, DOMAIN_NAME, CURRENT_PROFILE, PUPPET_MASTER_VERSION, SLEEP4SV
```
Comment (bogdando) [inline fuel_test/cobbler/vm_test_case.py:11]: add docs please!

**Pull request merged**